### PR TITLE
blade start

### PR DESCRIPTION
**Pull request by NickGhezzi**
started blade
+animation
+blade asset
+blade script
+test enemy damage script. needs to be replaced with the code that the gun uses

### CODE REVIEW

**Review by LeandroCCplusplus — APPROVED** [review]
Looks great, thanks babe ;)

File: CAPSTONE/Assets/Programming/Scripts/ALTPlayerController.cs [hunk 1]
```diff
@@ -51,6 +51,16 @@ public bool CheckForUseEquipmentInput()
...
 
+    public bool CheckForAttackPushed()
+    {
+        return Input.GetButtonDown("Fire1");
+    }
+
+    public bool CheckForAttackReleased()
+    {
+        return Input.GetButtonUp("Fire1");
+    }
```
Comment (adaless5) [inline CAPSTONE/Assets/Programming/Scripts/ALTPlayerController.cs:62]: Blade is an equipment and should use the same input as other Equipment

File: CAPSTONE/Assets/Programming/Scripts/Blade.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    {
+        if(playerController.CheckForAttackPushed())
+        {
+            animation.SetBool("attacking", true);
+            hitbox.enabled = true;
+        }
+        else if(playerController.CheckForAttackReleased())
+        {
+            animation.SetBool("attacking", false);
+            hitbox.enabled = false;
```
Comment (adaless5) [inline CAPSTONE/Assets/Programming/Scripts/Blade.cs:36]: This should use  CheckForUseEquipmentInput()

File: CAPSTONE/Assets/Programming/Scripts/MineSpawner.cs [hunk 3]
```diff
@@ -20,7 +20,7 @@ private void Start()
     // Update is called once per frame
     void Update()
     {
-        if (m_playerController.CheckForMineInput())
+        if (m_playerController.CheckForUseEquipmentInput())
         {
             if (m_bCanThrow)
             {
```
Comment (adaless5) [inline CAPSTONE/Assets/Programming/Scripts/MineSpawner.cs:26]: We'll create a CheckForUseWeaponInput() for Weapon tools

**Review by adaless5 — APPROVED** [review]
Aside from the changes I've commented, everything is good. I'll create a branch incorporating all this stuff.

**Pull request merged**